CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
  ### Changed
 - use caffeine cache instead of ehcache as ehcache has unnecessary blocking([#1007](https://github.com/ehrbase/ehrbase/pull/1007))
 - remove unnecessary DB queries([#1007](https://github.com/ehrbase/ehrbase/pull/1007))
+- Acquire the template identification only instead of composition.([#1042](https://github.com/ehrbase/ehrbase/pull/1042))
```

Comment:
Seems overly specific , maybe just mention performance improvements of the audit feature?

Reply:
done